tools/tools.go

```diff
@@ -0,0 +1,11 @@
+// +build tools
+
+package tools // import "gopkg.in/reform.v1/tools"
+
+import (
+	_ "github.com/AlekSi/gocoverutil"
+	_ "github.com/golangci/golangci-lint/cmd/golangci-lint"
+	_ "github.com/quasilyte/go-consistent"
+	_ "github.com/reviewdog/reviewdog/cmd/reviewdog"
+	_ "golang.org/x/tools/cmd/goimports"
```

Comment:
So, you specifically define some tools for limiting, but nor all of them (e.g. not depguard), why?

Reply:
Tools here are commands that will be build into `bin/`. depguard and other dependencies are already have versions defined: https://github.com/golangci/golangci-lint/blob/master/go.mod

Reply:
Ah, got it! Thanks!